app/views/content/life-as-a-teacher/age-groups-and-specialisms/secondary/_article.html.erb

```diff
@@ -0,0 +1,89 @@
+    <h2 class="heading--box-blue">Why teach <%= @front_matter["subject"] %></h2>
+    <div class="inset">
+      <p>Becoming a secondary school teacher is a great career choice if you want to make a difference in young people's lives. As students progress through school, you will directly influence their future prospects through the grades you help them achieve and the values you share with them.</p>
+      <p>As a specialist, you'll inspire students to develop a passion for your subject. You'll give them the skills and knowledge needed to tackle increasingly complex content and motivate some of them to pursue your subject beyond school.</p>
+      <!-- Why teach: benefits -->
+      <%= render 'content/shared/teaching/benefits' %>
+    </div>
+      <section class="col col-720">
+    <%= render 'content/shared/quotes/quote_art_richard' %>
+    </section>
+    <h2 class="heading--box-blue">Key stages</h2>
+    <p>Pupils start secondary school in England when they’re 11. Throughout secondary school, pupils will be taught by subject specialists and will gain increasingly greater independence over their own learning.</p>
+    <p> There are 3 stages of secondary education.</p>
+    <h3>Key stage 3 (ks3)</h3>
+    <p>Ks3 includes years 7 to 9, with children aged 11 to 14. In ks3 pupils will be exposed to a broad curriculum. At the end of the ks3, pupils will have some choice over the subjects they want to continue studying.</p>
+    <p>In ks3, pupils will be taught a range of compulsary subjects including:</p>
+     <ul>
+       <li>English</li>
+       <li>maths</li>
+       <li>science</li>
+       <li>history</li>
+       <li>geography</li>
+       <li>modern foreign languages</li>
+       <li>design and technology</li>
+       <li>art and design</li>
+       <li>music</li>
+       <li>physical education</li>
+       <li>religious education</li>
+       <li>citizenship</li>
+       <li>computing</li>
+     </ul>
+    <h3>Key stage 4 (ks4)</h3>
+    <p>Key stage 4 (KS4) includes years 10 and 11, with pupils aged 14 to 16.</p>
+     <p>During ks4, most pupils work towards GCSEs or another national qualification. There are some subjects that are compulsary and some which pupils can choose to study.</p>
+     <p>Compulsary subjects in ks4 include:</p>
+     <ul>
+      <li>English</li>
+      <li>maths</li>
+      <li>science</li>
+      <li>computing</li>
+      <li>physical education</li>
+      <li>religious education</li>
+      <li>citizenship</li>
+     </ul>
+     <p>Pupils can then choose from a range of subjects offered by the school. Schools must offer at least one subject from these areas:</p>
+     <ul>
+      <li>arts</li>
+      <li>design and technology</li>
+      <li>humanties</li>
+      <li>modern foreign languages</li>
+      </ul>
+    <h3>Key stage 5</h3>
+    <p>Key stage 5 (KS5), includes years 12 and 13, with students aged 16 to 18 at secondary schools with sixth forms, or colleges.</p>
```

Comment:
As discussed, worth bigging this up a bit along the lines of pupils exploring chosen subjects in more detail, can go more in depth, they're likely to be figuring out what they want to do as a career at this time, etc just getting across that it's an exciting time